src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -105,9 +105,11 @@ void addContextProviderEntity(ContextElementResponseVector& cerV, EntityId* enP,
 *
 * addContextProviderAttribute -
 *
+* The limitReached parameter is to prevent the addition of new entities, which is needed in the case of the pagination
+* limit has been reached with local entities.
 *
 */
-void addContextProviderAttribute(ContextElementResponseVector& cerV, EntityId* enP, ContextRegistrationAttribute* craP, const ProvidingApplication& pa)
+void addContextProviderAttribute(ContextElementResponseVector& cerV, EntityId* enP, ContextRegistrationAttribute* craP, const ProvidingApplication& pa, bool limitReached)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/9101970658842d30fb0d1a0166d115e5d0358a8c